GUIDE.md

```diff
@@ -62,6 +62,10 @@ Clicking on the blue button will display the Data selection form for that catego
 
 From here there are two options; either select an existing Data item from the items listed (and confirm with the 'Use' button), or *upload* a new Data item. It's also possible to download an existing Data item, for use as a template or reference, by selecting it and then using the 'Download' button.
 
+![Projection - Uploading](imgs/upload00.png) 
+
+To upload a file, use the 'Choose File' button and select the desired data source. There is then an option to eithre give this data source a brand new name, or to use in an existing name in which case the version number of the existing data source will be incremented. Clicking 'Upload' will begin the process.
```

Comment:
Maybe break this into two sentences ? This sentence is nearly 60 words long, and I had trouble following all of it the first time. Breaking it into two smaller ones is likely to make it easier to comprehend.

I'm basing this recommendation on this post here, about keeping sentences short, ideally 25 words or less.

https://insidegovuk.blog.gov.uk/2014/08/04/sentence-length-why-25-words-is-our-limit